adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenRequestTest.java

```diff
@@ -302,7 +302,7 @@ public void testFavorLocalCacheUseLocalRTSucceeds() throws PackageManager.NameNo
         final CountDownLatch signal = new CountDownLatch(1);
         signal.await(ACTIVITY_TIME_OUT, TimeUnit.MILLISECONDS);
 
-        // verify getAuthToken called once
+        // verify getAuthToken not called
```

Comment:
addressed?

---
In reply to: [70025700](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/676#discussion_r70025700) [](ancestors = 70025700)